Junk4Dollars-appTests/ShowAuctions/AuctionListTableViewDataSourceTests.swift

```diff
@@ -0,0 +1,23 @@
+import XCTest
+import UIKit
+import Junk4Dollars_app
+
+class AuctionListTableViewDataSourceTests: XCTestCase {
+    var dataSource: UITableViewDataSource!
+
+    func setupDataSource() {
+        dataSource = AuctionListTableViewDataSource()
+    }
+
+    func testNumberOfRowsHardcodedToThree() {
```

Comment:
How can you make these 2 tests slightly less hardcoded, so that they don't need to change once you start using real auctions?
The behavior of the methods won't change, just how the auctions list is initialized, right?

Reply:
Yeah, totally. Eric had wanted me to explicitly make them test that things are hardcoded, so we'd know to update them when we remove the hardcoded values. I do see the value of making this test more generic so that it will work even once the API is hooked up. What do you think?

Reply:
Maybe an intermediate step (PR) could be to extract the hardcoding outside of this class and have it at the same level as the API call would be.
So you would have all the plumbing ready for just replacing the hardcoding with an actual API call.